spec/requests/packages_spec.rb

```diff
@@ -1061,4 +1061,125 @@
       end
     end
   end
+
+  describe 'creating a custom package' do
+    let(:update_headers) do
```

Comment:
Ruby convention is that imperative blocks are wrapped in `do`, side-effect free blocks, like this one are delimited by `{}`